Docs/sphinx_documentation/source/LinearSolvers.rst

```diff
@@ -380,17 +380,21 @@ Available choices are
 - :cpp:`MLMG::BottomSolver::petsc`: Currently for cell-centered only.
 
 - :cpp:`LPInfo::setAgglomeration(bool)` (by default true) can be used
-  continue to coarsen the multigrid by copying what would have been the
-  bottom solver to a new :cpp:`MultiFab` with a new :cpp:`BoxArray` with
-  fewer, larger grids, to allow for additional coarsening.
+  to copy the current level of multigrid data to fewer, larger
+  boxes. Two advantages of using this option is that the bottom solver will become
+  smaller, and communication overhead is reduced.
+
+- :cpp:`LPInfo::setAgglomerationGridSize(int)` (by default 8 in 1D, 16 in 2D, 32 in 3D)
```

Comment:
If it's a GPU build, it's 32 be default. If it's a CPU build, it's 32 in 1D, 16 in 2D and 8 in 3D.  (The order in the original commit is incorrect.)

Reply:
@ajnonaka -- do you want to go ahead and fix this PR then we can merge?